examples/cluster/plot_cluster_comparison.py

```diff
@@ -79,6 +79,9 @@
     "min_samples": 7,
     "xi": 0.05,
     "min_cluster_size": 0.1,
+    "allow_single_cluster": True,
```

Comment:
Primarily for the final "null" cluster demonstration. I figured that rather than having a specific option for that plot, it would be better to enable this behavior for all plots for consistency in the comparison.